src/styles/main.scss

```diff
@@ -1,7 +1,36 @@
-@import "utils";
 @import "fonts";
-@import "typography";
+@import "variables";
+
+// reset here
+h1,
+h2,
+h3,
+h4,
+p {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+  transition-duration: 0.6s;
```

Comment:
don't do this, write transition on the elements you use it

Reply:
ok, but I dont see any problem in this

Reply:
you may need other values for many elements

Reply:
I can rewrite it on other elements (but it is already fixed. I just discuss)